src/vm/jitinterface.cpp

```diff
@@ -9968,28 +9968,11 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
         MODE_PREEMPTIVE;
     } CONTRACTL_END;
 
-    void* result = NULL;
-
-    if (ppIndirection != NULL)
-        *ppIndirection = NULL;
-
-#ifndef CROSSGEN_COMPILE
-    JIT_TO_EE_TRANSITION();
-
-    MethodDesc* ftn = GetMethod(method);
-    _ASSERTE(ftn->IsNDirect());
-    NDirectMethodDesc *pMD = (NDirectMethodDesc*)ftn;
-
-    if (pMD->NDirectTargetIsImportThunk())
-    {
-    }
-    else
-    {
-        result = pMD->GetNDirectTarget();
-    }
+    // Not expected to work due to multicore and tiered JIT potentially needing
+    // to call managed cctors
+    _ASSERTE(FALSE);
 
-    EE_TO_JIT_TRANSITION();
-#endif // CROSSGEN_COMPILE
+    void* result = NULL;
 
     return result;
```

Comment:
Unless it is really difficult to remove or has some future need, I would rather see it removed entirely.

Reply:
Based on @jkotas's comment, the interface is kept in sync between .NET Core and Framework, so this is the best we can do.

Reply:
Yep. Just read that. github hide it. Thanks.